spec/index.bs

```diff
@@ -1442,6 +1479,16 @@ shader. Compressed texture formats are provided by extensions. Their naming
 should follow the convention here, with the texture name as a prefix. e.g.
 `etc2-rgba8unorm`.
 
+The <dfn dfn>texel block</dfn> is a single addressable element of an uncompressed texture, or a single
+compressed block of a compressed texture. The <dfn dfn>texel block size</dfn> of a {{GPUTextureFormat}} is
+the number of bytes to store one [=texel block=]. The [=texel block size=] of each {{GPUTextureFormat}} is a
+constant value except for {{GPUTextureFormat/"depth24plus"}} and {{GPUTextureFormat/"depth24plus-stencil8"}}.
+
+The <dfn dfn>texel block width</dfn> and <dfn dfn> texel block height </dfn> specifies the dimension of
+one [=texel block=]. [=texel block width=] is the number of pixels in each row of one [=texel block=],
+and [=texel block height=] is the number of texel rows in one [=texel block=]. The [=texel block width=]
+and [=texel block height=] of non-compressed texture formats are 1.
```

Comment:
as @Kangz noted, this isn't true for YUV

Reply:
Done